src/components/Header/authApiMockResponse.ts

```diff
@@ -0,0 +1,118 @@
+// The URL endpoint to fetch patron data.
+const patronApiUrl = "https://platform.nypl.org/api/v0.1/auth/patron/tokens/";
+const accessToken =
+  "eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJpc3MiOiJodHRwczpcL1wvd3d3Lm55cGwub3JnIiwic3ViIjoiNjM2NzAyOCIsImF1ZCI6ImFwcF9sb2dpbiIsImlhdCI6MTQ4MjE3NjQ3MCwiZXhwIjoxNDgyMTgwMDcwLCJhdXRoX3RpbWUiOjE0ODIxNzY0NzAsInNjb3BlIjoib3BlbmlkIG9mZmxpbmVfYWNjZXNzIGNvb2tpZSBwYXRyb246cmVhZCJ9.JO7VbOqCC7HyjRmeyHD4zM1Gl0JBk5RdxjAkCp0h6sfVe-xs5FyY7biYqs19k4dUY2DbFYR5IG3xYt9IdhqyMkSnJxtiCY36WN7X_e0eBF2T1_IWKGaBc4JlbroMj5_aNB5W4nQvclrdlb2mV38Q_HGAMUKe8DDeCmAHctEtqGppNl8DC7IvqkekRS_6zgQwsHHW5kJR-f7zUROi4fvFpdNR-I7J4VNWdFIOijb4vXFOOWRLzdY_GHLJdWvSgxhqzwkceA5BScCicAKeHYHo04vabNp5TvPXoR0ypULqTyGYsNnXnUmh2Mu46j3bcNTACEKS97FBx1IfwttBL1ARtQ";
+export const patronApiUrlWithToken = `${patronApiUrl}${accessToken}`;
+
+// The URL endpoint to refresh the access token.
+const authServerDomain = "https://login.nypl.org/auth";
+export const tokenRefreshLink = `${authServerDomain}/refresh`;
```

Comment:
You should be able to import `patronApiUrl`, `authServerDomain`, and `tokenRefreshLink` from `headerUtils.ts`.